### PR TITLE
Feature/procedural cleanup

### PR DESCRIPTION
**Pull request by denis-yuen**
WIP with documentation update and some simple community recommended tickets

Addresses
* https://github.com/ga4gh/tool-registry-service-schemas/issues/229
* https://github.com/ga4gh/tool-registry-service-schemas/issues/114

### CODE REVIEW

**Comment by denis-yuen** [comment]
Note to self: I also tried to fix the 2.0.1 documentation builds which seem troubled.  It looks like documentation builds on tags as opposed to branches seems broken. 

Looks possibly like https://github.com/ga4gh/gh-openapi-docs/issues/43 but the fix is unreleased in the develop branch if I understand correctly. In the meantime, I created a branch https://github.com/ga4gh/tool-registry-service-schemas/tree/release-2.0.1 so that documentation for that release shows up in the documentation table of contents

**Comment by denis-yuen** [comment]
Note for self: need to put back swagger.json link in addition to openapi link to get both since we go "back in time" when generating a TOC

File: README.md [hunk 1]
```diff
@@ -1,51 +1,50 @@
...
 The [Global Alliance for Genomics and Health](http://genomicsandhealth.org/) (GA4GH) is an international
 coalition, formed to enable the sharing of genomic and clinical data.
 
-The GA4GH [Data Working Group](http://ga4gh.org/#/) concentrates on data representation, storage,
-and analysis, including working with platform development partners and
-industry leaders to develop standards that will facilitate
-interoperability.
-
-Containers and Workflows Task Team
+Cloud Work Stream
```
Comment (coverbeck) [inline README.md:18]: [This page](https://www.ga4gh.org/work_stream/cloud/) spells it `Workstream`.

File: README.md [hunk 2]
```diff
@@ -1,51 +1,50 @@
...
 ----------------------------------
 
-The Containers & Workflows working group is an informal, multi-vendor working group born out of the BOSC 2014 codefest, consisting of various organizations and individuals that have an interest in portability of data analysis workflows. Our goal is to create specifications that enable data scientists to describe analysis tools and workflows that are powerful, easy to use, portable, and support reproducibility for a variety of problem areas including data-intensive science like bioinformatics, physics, and astronomy; and business analytics such as log analysis, data mining, and ETL.
+The Cloud Work Stream is focused on creating specific standards for defining, sharing, and executing portable workflows and accessing data across clouds.
```
Comment (coverbeck) [inline README.md]: Workstream

File: README.md [hunk 3]
```diff
@@ -1,51 +1,50 @@
...
-  depending on the tool/workflow submitter
-* ID:  globally unique across systems and also identifies the system that it came from (ex: 123456323@agora.broadinstitute.org )
-
+* Serve up one or more of CWL, WDL, Nextflow, Galaxy or Snakemake to describe a tool or represent a workflow depending on the tool/workflow submitter 
+* Download individual workflow files or the entire workflow as a zip
+* Interrogate the language versions of these workflows
+* Get specific versions of workflows and tools, potentially with immutable versions with checksums on their files
+* ID: globally unique across systems and also identifies the system that it came from (ex: 123456323@agora.broadinstitute.org )
+* Refer to our [data model](https://ga4gh.github.io/tool-registry-service-schemas/DataModel/) for information on how to define TRS URIs if desired
+* provides more structure than a simple un-formatted list of tools but it is also a standard for registries to implement as opposed to a registry implementation itself
```
Comment (coverbeck) [inline README.md]: Previous bullet items start with capital letter

Comment (coverbeck) [reply README.md]: No hyphen in `un-formatted`, at least not here: https://www.merriam-webster.com/dictionary/unformatted

File: README.md (hunk 1 above)
Comment (denis-yuen) [reply README.md:18]: Hmmm, https://github.com/ga4gh/wiki/wiki does it the other way. Will check in meeting

Comment (denis-yuen) [reply README.md:18]: From call, it seemed like the preference was "Work stream" with a space as in https://www.ga4gh.org/how-we-work/workstreams/

**Review by coverbeck — APPROVED** [review]

**Review by uniqueg — APPROVED** [review]

**Pull request merged**